util/pom.xml

```diff
@@ -86,6 +91,11 @@
             <groupId>com.samskivert</groupId>
             <artifactId>jmustache</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.sun.jersey</groupId>
```

Comment:
Similar remark: Kill Bill itself is a library at its core and doesn't depend on `com.sun.jersey`. Very large companies, which have their own in-house Java container, don't use the Kill Bill profiles, but instead wire up the Kill Bill library jars.